recipes/openssl/1.x.x/conanfile.py

```diff
@@ -329,6 +330,7 @@ def _targets(self):
             "Macos-ppc32be-*": "%sdarwin-ppc-cc" % self._target_prefix,
             "Macos-ppc64-*": "darwin64-ppc-cc",
             "Macos-ppc64be-*": "darwin64-ppc-cc",
+            "Macos-armv8-*": "darwin64-arm64-cc" if has_darwin_arm else "darwin-common",
```

Comment:
@SpaceIm 
I think you have an ARM Apple.
Can you verify this pr?

Reply:
Nop, Apple Intel. I can test cross-build to Apple M1 like CI already does.